### DISCUSSION TITLE
Can we omit build directory in distribution of socket.io-client?

### DISCUSSION THREAD

**Discussion by riesel** [socketio/socket.io#4179] [Q&A] [answered]
We are including the socket.io-client package from https://www.npmjs.com/package/socket.io-client in our vue.js frontend.
There we are using [webpack-license-plugin ](https://www.npmjs.com/package/webpack-license-plugin) to scan for licenses in dependencies in our project. 
This worked fine for socket.io-client versions < v4.4.0. From 4.4.0 on we see that the content of package.json in e.g. build\csj\  directory changed and now contains name and version information. Since it does not contain license information, webpack-license-plugin reports a missing license.

My question is whether the build directory has to be packaged at all or can be omitted to avoid this issue?

**Comment by darrachequesne** [comment, marked as answer]
Zut. I guess we'll have to add the license too...

See also: https://github.com/socketio/socket.io-client/issues/1513 (fixed by https://github.com/socketio/socket.io-client/commit/53d8fcafabbfddb5834012c9c98743bfe6e13347)

**Reply by riesel** [reply]
Yes, that would fix the problem. Basically 
- license info and ideally repository info in package.json
- license file in same directory

**Reply by jschirrmacher** [reply]
Why is it necessary to include the `/build` folder in the distributed package?
